runtime/syntax/nftables.yaml

```diff
@@ -1,19 +1,23 @@
 filetype: nftables
 
 detect:
-    filename: "nftables.conf$"
-    header: "^(#!.*/(env +)?nft( |$)|flush +ruleset)"
-    
+   filename: "(nftables\\.(conf|rules)$|nftables(\\.rules)?\\.d/)"
+   header: "^(#!.*/(env +)?nft( |$)|flush +ruleset)"
+
```

Comment:
The indentation level changes from 4 to 3, but why?

Reply:
That has no reason. The Main Reason is that i edited it with multiple editors, and in one editor was tab set to 3 spaces and in the other one to 4.